### DISCUSSION TITLE
Welcome to E-commerce-project-springBoot Discussions!

### DISCUSSION THREAD

**Discussion by jaygajera17** [jaygajera17/E-commerce-project-springBoot#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AmnesiaZero** [comment]
Hi,why don't you use lombok to get rid of all setters/gettrs and make code more clear?

**Comment by jaygajera17** [comment]
Never heard about Lombok ,thanks for your suggetion i’ll explore. If you already made code clear then send pull request. From: AmnesiaZeroSent: 17 August 2023 14:03To: jaygajera17/E-commerce-project-springBootCc: Jay Gajera; AuthorSubject: Re: [jaygajera17/E-commerce-project-springBoot] Welcome to E-commerce-project-springBoot Discussions! (Discussion #20) Hi,why don't you use lombok to get rid of all setters/gettrs and make code more clear?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by jaygajera17** [comment]
@NisargPipaliya , labels check are failing in the PR , can you please create a steps or guide for better understanding to contributer.

**Reply by NisargPipaliya** [reply]
Okay,
I will,
let be get some time
